src/main/scala/cromwell/binding/WdlExpression.scala

```diff
@@ -233,6 +252,10 @@ object WdlExpression {
         val lhs = toString(a.getAttribute("lhs"), highlighter)
         val rhs = toString(a.getAttribute("rhs"), highlighter)
         s"$lhs.$rhs"
+      case a: Ast if a.isArrayOrMapIndex =>
+        val lhs = toString(a.getAttribute("lhs"), highlighter)
+        val rhs = toString(a.getAttribute("rhs"), highlighter)
+        s"$lhs[$rhs]"
     }
```

Comment:
Looks like a lot of cases are missing here - e.g. ArrayLiteral - what happens if toString is called on those?